flink-connector-mongodb-cdc/src/main/java/com/ververica/cdc/connectors/mongodb/source/config/MongoDBSourceOptions.java

```diff
@@ -133,4 +133,12 @@ public class MongoDBSourceOptions {
                     .defaultValue(64)
                     .withDescription(
                             "The chunk size mb of incremental snapshot. Defaults to 64mb.");
+
+    @Experimental
+    public static final ConfigOption<Boolean> FULL_DOCUMENT_PRE_POST_IMAGE =
+            ConfigOptions.key("full.document.pre.post.image")
```

Comment:
`document`, `pre-image` and `post-image` are MongoDB concepts, when defined this option in Flink Connector/SQL, I prefer to use changelog related concept for Flink users, how about `scan.full.changelog` or something else?

Reply:
Renamed option to `scan.full.changelog` / `.scanFullChangelog`.

Reply:
`scan.full-changelog` may be better when compared to other flink connector options

Reply:
Done.